### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#63226] [Other Feature Feedback, Questions, & Ideas] [answered]
Can someone tell me what's wrong with my website? Its not getting stylesheet, i checked the links, file paths and names, everything looks correct but it still doesn't work

[Bug here](https://github.com/Nit-nit/Nit-nit.github.io/blob/main/index.html)

**Comment by nirav-gajera** [comment, marked as answer]
1. in your index.html  add : `type="text/css"`

    `<link rel="stylesheet" href="/css/style.css" type="text/css" />`
 (your code is ` href="/css/styles.css"` and file name is `style` correct that)
2. Inspect Network Requests: Open your browser's developer tools,`ctrl+shift+i` go to the "Network" tab, and check if the styles.css file is being requested. If it's not, there might be an issue with the file path or server configuration. 

3. Server Status: If you're using a server, ensure that it's running and accessible. You can try accessing the CSS file directly in your browser by  direct entering the URL to the CSS file (e.g., http://localhost/css/styles.css).

**Reply by NitinIsFit** [reply]
Damn man, how didn't I find that typo even after trying to find it out many many times, thanks man, i really really appreciate your efforts and help 🙏

**Reply by nirav-gajera** [reply]
Small Changes Make Big Difference
Happy coding!

**Comment by mxritzdev** [comment]
in your head tag, add a `<base href="/">` tag.

**Reply by NitinIsFit** [reply]
What will that do?

**Reply by nirav-gajera** [reply]
The <base> element specifies the base URL to use for all relative URLs in a document. 

Links pointing to a fragment in the document  e.g.` <a href="#some-id">`  are resolved with the `<base>`, triggering an HTTP request to the base URL with the fragment attached.

For example, given` <base href="https://example.com/">` and this link: `<a href="#anchor">To anchor</a>`. The link points to `https://example.com/#anchor.`
